app/uk/gov/hmrc/cgtpropertydisposalsfrontend/services/onboarding/OnboardingAuditService.scala

```diff
@@ -36,8 +36,8 @@ import uk.gov.hmrc.play.audit.model.ExtendedDataEvent
 
 import scala.concurrent.ExecutionContext
 
-@ImplementedBy(classOf[AuditServiceImpl])
-trait AuditService {
+@ImplementedBy(classOf[OnboardingAuditServiceImpl])
+trait OnboardingAuditService {
```

Comment:
this has stuff to do with accounts in it

Reply:
This needs to be overhauled and will be done when the phase 2 work starts